app/models/site_customization/page.rb

```diff
@@ -6,11 +6,13 @@ class SiteCustomization::Page < ActiveRecord::Base
                    format: { with: /\A[0-9a-zA-Z\-_]*\Z/, message: :slug_format }
   validates :title, presence: true
   validates :status, presence: true, inclusion: { in: VALID_STATUSES }
-  validates :locale, presence: true
-
+  
+  translates :title, :subtitle, :content
+  globalize_accessors locales: [:en, :es, :fr, :nl, :val, :pt_br] 
+ 
   scope :published, -> { where(status: 'published').order('id DESC') }
   scope :with_more_info_flag, -> { where(status: 'published', more_info_flag: true).order('id ASC') }
-  scope :with_same_locale, -> { where(locale: I18n.locale).order('id ASC') }
+#  scope :with_same_locale, -> { where(locale: I18n.locale).order('id ASC') }
```

Comment:
Can you remove the commented-out code? No need to keep it around :)

Reply:
Ok!